src/main/resources/META-INF/rewrite/testcontainers.yml

```diff
@@ -0,0 +1,118 @@
+      image: "confluentinc/cp-kafka:5.4.3"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.localstack.LocalStackContainer
+      image: "localstack/localstack:0.11.2"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.MariaDBContainer
+      image: "mariadb:10.3.6"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.MockServerContainer
+      image: "jamesdbloom/mockserver:mockserver-5.5.4"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.MongoDBContainer
+      image: "mongo:4.0.10"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.MSSQLServerContainer
+      image: "mcr.microsoft.com/mssql/server:2017-CU12"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.MySQLContainer
+      image: "mysql:5.7.34"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.Neo4jContainer
+      image: "neo4j:4.4"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.NginxContainer
+      image: "nginx:1.9.4"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.OracleContainer
+      image: "gvenzl/oracle-xe:18.4.0-slim"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.OrientDBContainer
+      image: "orientdb:3.0.24-tp3"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.PostgreSQLContainer
+      image: "postgres:9.6.12"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.PulsarContainer
+      image: "apachepulsar/pulsar:2.10.0"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.RabbitMQContainer
+      image: "rabbitmq:3.7.25-management-alpine"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.SolrContainer
+      image: "solr:8.3.0"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.containers.ToxiproxyContainer
+      image: "shopify/toxiproxy:2.1.0"
+  - org.openrewrite.java.testing.testcontainers.ExplicitContainerImage:
+      containerClass: org.testcontainers.vault.VaultContainer
+      image: "vault:1.1.3"
```

Comment:
These versions seem to line up well with what I found through `grep -r 'DEFAULT_TAG = ' modules/ | cut --delimiter=" " --fields=1,11 | sort` on testcontainers/testcontainers-java:
https://github.com/search?q=repo%3Atestcontainers%2Ftestcontainers-java%20%22DEFAULT_TAG%20%3D%20%22&type=code

There's been some new addition though such as PostGIS, but I haven't checked each of them to see if those new additions contain a default no arg constructor. As-is this is already valuable I suppose. New ones are also easily added after a merge.

Reply:
we are making sure new ones don't provide a default constructor. A recommended practice is to declare the image and tag to use.

All versions are good 👍🏽 I compared to the [initial implementation](https://github.com/openrewrite/rewrite-testcontainers/blob/main/src/main/resources/META-INF/rewrite/testcontainers.yml)